server/text_generation_server/layers/gptq/__init__.py

```diff
@@ -332,6 +336,7 @@ def get_weights_row(self, weights: Weights, prefix: str):
                     # Exllama implementation does not support row tensor parallelism with act-order, as
                     # it would require to reorder input activations that are split unto several GPUs
                     use_exllama = False
+                    desc_act = True
```

Comment:
I don't know why this ignores the value of `desc_act` given by the configuration, isn't that the source of truth? Do we expect some models to use activation sorting but lying about it in the config? Needs a comment.

Reply:
ack